applications/vessel-history/src/features/map/info/Info.module.css

```diff
@@ -0,0 +1,118 @@
+.infoContainer {
+  width: 80%;
```

Comment:
We should use units in `rem` instead of `px` so that the sizes and spaces adjusts better.